src/main/ts/component/Editor.ts

```diff
@@ -48,6 +48,7 @@ const parseNumberOrString = (value: string) => /^\d+$/.test(value) ? Number.pars
 
 const configAttributes: Record<string, (v: string) => unknown> = {
   setup: parseGlobal, // function
+  statusbar: parseBooleanOrString, // boolean
```

Comment:
Should this be `// boolean or string`

Reply:
I thought the same, but it is a boolean: https://www.tiny.cloud/docs/tinymce/latest/statusbar-configuration-options/#statusbar

I think we both confused with toolbar

Reply:
Hmm, then `parseBooleanOrString` should be `parseBoolean` ?

Reply:
I used that because is what is used for other fields like `powerpaste_allow_local_images` and `images_upload_credentials`, maybe I'm wrong but the attribute will be always a string so even `true` and `false` will be `"true"` and `"false"`

Reply:
Right since they are all strings it needs to check for specific values or fallback to the string value. Makes sense now. I really should have spent time to look thought the webcomponent plugin this sprint but all other stuff came in between.